app/web/template/layout/people-to-notify-summary.gohtml

```diff
@@ -0,0 +1,62 @@
+{{ define "people-to-notify-summary" }}
+    {{range $i, $p := .PeopleToNotify}}
+        {{ $personNumber := inc $i }}
+
+        <h2 class="govuk-heading-m">{{ trFormat $.App "personCount" "PersonNumber" $personNumber }}</h2>
```

Comment:
this header looks weird, if we are keeping it then it should probably be smaller and lower priority than the "People to notify" section header

Reply:
@hawx are you referring to the physical font size with this or the hierarchical `h` elements? The markup looks to be in order:

<img width="497" alt="Screenshot 2022-12-15 at 10 43 45" src="https://user-images.githubusercontent.com/17926619/207840010-dd0d1705-2296-4183-9d30-b1e95b2818d4.png">

And the physical font size looks like its comparatively less prominent than the page header:

<img width="476" alt="Screenshot 2022-12-15 at 10 43 52" src="https://user-images.githubusercontent.com/17926619/207841675-624dcf7f-e74f-4e61-9390-7e600cc07fd0.png">